src/config/review-providers.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Review provider configurations
+    |--------------------------------------------------------------------------
+    */
+    'reviews_io' => [
+        'store' => env('REVIEWS_CO_UK_STORE'),
+        'api_key' => env('REVIEWS_CO_UK_API_KEY'),
+        'api_base_host' => env('REVIEWS_CO_UK_API_BASE_HOST'),
```

Comment:
I think we should change the names of these values to match the api documentation, what do you think?

![image](https://user-images.githubusercontent.com/58700660/95446220-e6910300-0957-11eb-8d83-7c01b11e20f0.png)

small point but will help understanding by new dev in future

Reply:
'api_url' instead of 'api_base_host'
'store_id' instead of 'store'

I think it's more important that the config value names match the api documentation names than that they match each other, but happy to have my mind changed here :+1: